app-leechcraft/lc-aggregator/lc-aggregator-0.6.75_pre20200510.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit leechcraft
+
+DESCRIPTION="Full-featured RSS/Atom feed reader for LeechCraft"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug mysql postgres +sqlite"
+
+DEPEND="
+	~app-leechcraft/lc-core-${PV}[postgres?]
+	dev-qt/qtcore:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsql:5
+	dev-qt/qtwebkit:5
```

Comment:
(Just FYI Gentoo is trying to get rid of this, is it possible to use qtwebengine upstream?)